prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -1339,28 +1360,19 @@ struct rxr_rx_entry *rxr_pkt_alloc_rtw_rx_entry(struct rxr_ep *ep,
 	return rx_entry;
 }
 
-void rxr_pkt_handle_eager_rtw_recv(struct rxr_ep *ep,
-				   struct rxr_pkt_entry *pkt_entry)
+void rxr_pkt_proc_eager_rtw_recv(struct rxr_ep *ep,
```

Comment:
The function name ```rxr_pkt_proc_eager_rtw_recv``` is a little redundant, ```rxr_pkt_proc_eager_rtw``` should be enough.

Reply:
I'll change it.

Reply:
Function name changed to `rxr_pkt_proc_eager_rtw `, mark as resloved.